data/json/terrain.json

```diff
@@ -3156,9 +3156,29 @@
         "id" : "t_tree_maple",
         "name": "maple tree",
         "symbol": "7",
-        "color": "red",
+        "color": [ "red", "light_green", "light_green", "red" ],
         "move_cost": 0,
-        "flags": ["FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT"],
+        "flags": [ "FLAMMABLE_ASH", "NOITEM", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+        "examine_action": "tree_maple",
+        "bash": {
+            "str_min": 80, "str_max": 180,
+            "sound": "crunch!",
+            "sound_fail": "whack!",
+            "ter_set": "t_dirt",
+            "items": [
+                { "item": "stick", "count": [3, 10] },
+                { "item": "splinter", "count": [10, 25] }
+            ]
+        }
+    },{
+        "type" : "terrain",
+        "id" : "t_tree_maple_tapped",
+        "name": "maple tree",
+        "symbol": "7",
+        "bgcolor": "light_cyan",
+        "move_cost": 0,
+        "flags": [ "FLAMMABLE_ASH", "SUPPORTS_ROOF", "TREE", "REDUCE_SCENT" ],
+        "examine_action": "tree_maple_tapped",
```

Comment:
Also walnuts apparently. We don't have walnuts but we could have them.

Though I was talking more about the whole approach. Duplicating terrain isn't wrong, but the fact that taped tree has different flags than the untaped one could lead to subtle bugs. On the other hand, it's easier to verify whether a tree is tapped or not.

Probably the best solution would be to duplicate the funnel approach, but that's for later on, if someone decides to add more saps.